examples/disaggregated/slurm/benchmark/submit.py

```
@@ -631,7 +710,7 @@ def submit_job(config, log_dir, dry_run):
     client_slurm_prefix = [
         f"srun -l --container-name={container_name}",
         f"--container-mounts={container_mount_str}",
-        f"--mpi=pmix --overlap -N 1 -n 1",
+        f"--no-container-mount-home --mpi=pmix --overlap -N 1 -n 1",
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)